zuora-api/payment-methods.js

```diff
@@ -10,6 +10,8 @@ module.exports = {
   // https://rest.zuora.com/v1/payment-methods/credit-cards
   create: requestOptions => request('POST', 'payment-methods/credit-cards', requestOptions),
 
+  decryption: requestOptions => request('POST', 'payment-methods/decryption', requestOptions),
```

Comment:
you mind adding this as a comment 

`https://zuora-api-reference-3.api-docs.io/2018-07-05/v1/create-apple-pay-payment-method`